android/src/main/java/com/mapbox/mapboxgl/MapboxMapController.java

```diff
@@ -228,14 +228,7 @@ private CameraPosition getCameraPosition() {
   }
 
   private SymbolBuilder newSymbolBuilder() {
```

Comment:
Since this isn't really doing anything relevant anymore, couldn't we just replace its usage(s) with `new SymbolBuilder()`?

Reply:
Sure! Sorry for delay. I have updated PR.